src/OrbitGl/include/OrbitGl/Batcher.h

```diff
@@ -32,20 +33,29 @@ class Batcher : public BatcherInterface {
     uint32_t draw_calls = 0;
     uint32_t stored_layers = 0;
     uint32_t stored_vertices = 0;
+
+    friend bool operator==(const Batcher::Statistics& lhs, const Batcher::Statistics& rhs);
+    friend bool operator!=(const Batcher::Statistics& lhs, const Batcher::Statistics& rhs);
   };
 
   [[nodiscard]] virtual Statistics GetStatistics() const = 0;
-
-  friend bool operator==(const Batcher::Statistics& lhs, const Batcher::Statistics& rhs);
-  friend bool operator!=(const Batcher::Statistics& lhs, const Batcher::Statistics& rhs);
+  [[nodiscard]] std::string GetCurrentRenderGroupName() const override {
+    return current_render_group_.name;
+  }
+  void SetCurrentRenderGroupName(std::string name) override {
+    current_render_group_.name = std::move(name);
+  }
 
  protected:
   orbit_gl::TranslationStack translations_;
+  BatchRenderGroupId current_render_group_;
 
  private:
   BatcherId batcher_id_;
 };
 
+[[nodiscard]] bool operator==(const Batcher::Statistics& lhs, const Batcher::Statistics& rhs);
```

Comment:
Hmmm. Is this needed?

Reply:
Actually it's not required at all. I've removed it.